features/20-standalone-launch.feature

```diff
@@ -31,6 +31,14 @@ Feature: Follows the SMART standalone launch sequence
         When I ask for authorization without the state field
         Then the response code should not be 200
 
+    Scenario: EHR evaluates authorization request - very long "state"
```

Comment:
I don't know if this is actually something we want to test for, but I wanted to put together the scaffolding for testing the full authorization process, and this seemed like an easy thing to throw in there.